notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,2380 @@
+{
```

Comment:
<p>About this, I'm wondering if the steps we have to follow are small enough so we can list them in this notebook, so the user doesn't have to go to another place. </p><p>Could you please tell me what are the exact steps that I would have to follow to set up the workspace?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/681/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
As discussed in a separate thread, we're going to pull in workspace set up here

Reply:
Used the same %load as PR 575